pages/index.tsx

```diff
@@ -71,24 +71,24 @@ const Home: NextPage = () => {
 
 	useEffect(() => {
 		const getUserStaked = async () => {
-			const userStakedToken = await near.nearViewFunction({
-				contractName: CONTRACT.FARM,
+			const userStakedToken = await viewFunction({
+				receiverId: CONTRACT.FARM,
 				methodName: `list_user_seeds`,
 				args: {
 					account_id: accountId,
 				},
 			})
 
-			const userStakedNFTData = await near.nearViewFunction({
-				contractName: CONTRACT.FARM,
+			const userStakedNFTData = await viewFunction({
+				receiverId: CONTRACT.FARM,
 				methodName: `list_user_nft_seeds`,
 				args: {
 					account_id: accountId,
 				},
 			})
 
-			setUserStaked(userStakedToken)
-			setUserStakedNFT(userStakedNFTData)
+			setUserStaked(userStakedToken as IUserStaked)
+			setUserStakedNFT(userStakedNFTData as IUserStakedNFT)
 		}
```

Comment:
instead of lu `as` in return sebenernya bisa gini dikasih <> di setelah viewFunction
```js
// better gini
const userStakedNFTData = await viewFunction<IUserStakedNFT>({
	receiverId: CONTRACT.FARM,
	methodName: `list_user_nft_seeds`,
	args: {
		account_id: accountId,
	},
})
setUserStakedNFT(userStakedNFTData)

// daripada
setUserStakedNFT(userStakedNFTData as IUserStakedNFT)
```

kalo mau baca2 lebih detail bisa disini, atau googling keywordnya generic type
https://www.typescriptlang.org/docs/handbook/2/generics.html

Reply:
kayaknya semua viewFunction bisa diginiin juga

Reply:
wokeh hu thanks hu